README.md

```diff
@@ -1,2 +1,71 @@
-# FIRST-MOBILE-PORTFOLIO-
-This is my first mobile portfolio constructed at the microverse program
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# First Mobile Portfolio
+
+> Description of the project.
+
+
+## Built With
+
+- css
+- html
+- linters
+
+## Live Demo ( Not available)
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
Kindly add the link to live version here instead of adding it down in the deployment section 🤗